features/steps/cocoa_steps.rb

```diff
@@ -193,49 +193,53 @@ def check_device_model(field, list)
 end
 
 Then('the error is valid for the error reporting API') do
-  case Maze.driver.capabilities['platformName']
-  when 'iOS'
+  platform = Maze::Helper.get_current_platform
+  STDOUT.puts "platform: #{platform}"
```

Comment:
Is this a debugging log statement that should be removed?

Reply:
Yes, you're right - good spot.